docs/source/cloud_startup.rst

```diff
@@ -0,0 +1,90 @@
+============================================
+Cloud Startup
+============================================
+
+Overview
+********
+
+This page provides the steps necessary to get going with data ingestion and
+querying of the PDS Registry Service on the cloud from the perspective of a Discipline
+Node (DN).
+
+Initial set-up
+**************
+
+To get things going, you should have provided the EN contact with the following:
+
+  * A list of user email addresses which will be authorized to perform data ingestion.
+  * A list of IP addresses or blocks of addresses from which data ingestion will occur.
+
+Opensearch accounts will be set up for each user and their username and (temporary)
+password will be sent to them via JPL Large File Transfer (LFT) which provides secure
+transit of the credentials. The IP addresses are added to the Opensearch access whitelist.
+
+Along with your Opensearch user credentials you will receive the URL which points to your Opensearch
+instance - we will refer to this as the Opensearch URL.
+
+Changing Your Opensearch password
```

Comment:
[Title Case ](https://en.wikipedia.org/wiki/Title_case)Suggestion: **Changing Your Opensearch Password**

Reply:
Done, I first thought you missed it so I corrected and then I saw your comment !